private/tools/java/rules/jvm/external/maven/MavenPublisher.java

```diff
@@ -89,6 +89,8 @@ public static void main(String[] args)
     Path binJar = getPathIfSet(args[6]);
     Path srcJar = getPathIfSet(args[7]);
     Path docJar = getPathIfSet(args[8]);
+    String extraClassifiers = args.length > 9 ? args[9] : null;
```

Comment:
Maybe default to the empty string or add a dep on guava and use `Strings.isNullOrEmpty` later on.

Reply:
will do